src/dlgRoomProperties.cpp

```diff
@@ -0,0 +1,555 @@
+        itSymbolUsed.next();
+        symbolCountsSet.insert(itSymbolUsed.value());
+    }
+
+    // Obtains a list of those values sorted in ascending count of use
+    QList<int> symbolCountsList{symbolCountsSet.begin(), symbolCountsSet.end()};
+    if (symbolCountsList.size() > 1) {
+        std::sort(symbolCountsList.begin(), symbolCountsList.end());
+    }
+
+    // Build a list of strings for display in descending count of use
+    QStringList displayStrings;
+    displayStrings.append(multipleValuesPlaceholder);
+    for (int i = symbolCountsList.size() - 1; i >= 0; --i) {
+        itSymbolUsed.toFront();
+        while (itSymbolUsed.hasNext()) {
+            itSymbolUsed.next();
+            if (itSymbolUsed.value() == symbolCountsList.at(i)) {
+                displayStrings.append(qsl("%1 {%2:%3}")
+                    .arg(itSymbolUsed.key())
+                    .arg(tr("count",
+                            // Intentional comment to separate arguments
+                            "This text will be part of a list of room values shown, which will show the value "
+                            "itself, followed by the counted number of rooms with this very value like: "
+                            "grey {count:2} - so please translate like counted ammount, number of, etc."))
+                    .arg(QString::number(itSymbolUsed.value())));
+            }
+        }
+    }
+    return displayStrings;
+}
+
+
+QStringList dlgRoomProperties::getComboBoxWeightItems()
+{
+    // Obtain a set of "used" values
+    QHashIterator<int, int> itWeightUsed(mpWeights);
+    QSet<int> weightCountsSet;
+    while (itWeightUsed.hasNext()) {
+        itWeightUsed.next();
+        weightCountsSet.insert(itWeightUsed.value());
+    }
+
+    // Obtains a list of those values sorted in ascending count of use
+    QList<int> weightCountsList{weightCountsSet.begin(), weightCountsSet.end()};
+    if (weightCountsList.size() > 1) {
+        std::sort(weightCountsList.begin(), weightCountsList.end());
+    }
+
+    // Build a list of strings for display in descending count of use
+    QStringList displayStrings;
+    displayStrings.append(multipleValuesPlaceholder);
+    for (int i = weightCountsList.size() - 1; i >= 0; --i) {
+        itWeightUsed.toFront();
+        while (itWeightUsed.hasNext()) {
+            itWeightUsed.next();
+            if (itWeightUsed.value() == weightCountsList.at(i)) {
+                displayStrings.append(qsl("%1 {%2:%3}")
+                    .arg(QString::number(itWeightUsed.key()))
+                    .arg(tr("count",
+                            // Intentional comment to separate arguments
+                            "This text will be part of a list of room values shown, which will name the value "
+                            "itself, followed by the counted number of rooms with that very value like: "
+                            "grey {count: 2} - So please translate like counted amount, number of, etc."))
```

Comment:
This is a numeric entry so use a number not a word:
```suggestion
                            "42 {count: 2} - So please translate like counted amount, number of, etc."))
```

Reply:
Technically true, but by keeping the comment identical, this string will only presented to translators once, so this is deliberately done so.